docs/ref/models/expressions.txt

```diff
@@ -64,10 +64,11 @@ Some examples
     # Aggregates can contain complex computations also
     Company.objects.annotate(num_offerings=Count(F('products') + F('services')))
 
-    # Expressions can also be used in order_by()
+    # Expressions can also be used in order_by(), either directly
     Company.objects.order_by(Length('name').asc())
     Company.objects.order_by(Length('name').desc())
-
+    # or using the double underscore syntax.
+    Company.objects.order_by('products__count')
```

Comment:
Does `__count` need to be registered as a transform for this to work? I guess it's a bit unclear what `products` refers to since the `Company` model isn't defined as far as I see.

Reply:
A good point. I'm going to make this use `name__length` instead. It's a bit clearer, as name is a normal field rather than a relationship and we have a similar example in the 1.10 release notes.